extensions/vscode/src/views/homeView.ts

```diff
@@ -1379,10 +1382,15 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
           },
         });
       } catch (error: unknown) {
-        const summary = getSummaryStringFromError(
-          "sendRefreshedFilesLists, files.getByConfiguration",
-          error,
-        );
+        let summary = "";
+        if (isAxiosJsonErrorRes(error)) {
+          summary = error.response.data.message;
+        } else {
+          summary = getSummaryStringFromError(
+            "sendRefreshedFilesLists, files.getByConfiguration",
```

Comment:
These strings were added so that we could quickly identify where the code was failing. We should figure out a way to keep them, even if it is just to be used in log messages, rather than the display.